lib/reducers/ui.js

```diff
@@ -251,6 +253,7 @@ const reducer = (state = initial, action) => {
 
     case SESSION_SET_ACTIVE:
       state_ = state.merge({
+        contextMenuOpen: false,
```

Comment:
I'm not sure it is a good idea to maintain context menu state in renderer redux store. If you want to keep this, we must find a better way to pass it to `false`. Because I think that if you click in another application window, contextMenu will close but you wont have a `SESSION_SET_ACTIVE` redux action.